src/hooks/api/useNoteEsc.js

```diff
@@ -0,0 +1,53 @@
+import { useQuery } from 'react-query';
+import { useIntl } from 'react-intl';
+import { useMemo } from 'react';
+
+import { useNamespace } from '@folio/stripes/core';
+
+import { getMappedAndSortedNotes } from '../../utils/helpers';
+import { usePublishCoordinator } from '../usePublishCoordinator';
+
+const DEFAULT_DATA = {};
+
+export const useNotesEsc = (namespaceKey, tenants, type, categoryId, options = {}, url, noteKey, optionType, parameterKey) => {
+  const [namespace] = useNamespace({ key: namespaceKey });
+  const { initPublicationRequest } = usePublishCoordinator(namespace);
+  const { formatMessage } = useIntl();
+
+  const { data = DEFAULT_DATA, isFetching } = useQuery({
+    queryKey: [namespace, tenants, url, type],
+    queryFn: async () => {
+      const { publicationResults } = await initPublicationRequest({
+        url,
+        method: 'GET',
```

Comment:
Good practice could be to keep all such method names in constants

Reply:
done as well